config/build/buildconfig.yaml

```diff
@@ -21,51 +21,53 @@ objects:
 - apiVersion: build.openshift.io/v1
   kind: BuildConfig
   metadata:
-    name: producer-python
+    name: producer-quarkus
     namespace: ${IMAGE_NAMESPACE}
   spec:
     output:
       to:
         kind: ImageStreamTag
-        name: producer-python:latest
+        name: producer-quarkus:latest
         namespace: ${IMAGE_NAMESPACE}
     postCommit: {}
     source:
       git:
-        uri: https://github.com/dmartinol/knative-quickstarts.git
+        uri: https://github.com/JudeNiroshan/knative-quickstarts.git
```

Comment:
can you change it back to the main repo?

Reply:
4c5c0c7026d4cd4ccec117a12edd31eb0026743f